lib/l10n/app_en.arb

```diff
@@ -7,5 +7,6 @@
   "extremely_likely": "Extremely Likely",
   "start_survey": "Start Survey",
   "submit_survey": "Submit Survey",
-  "today": "Today"
+  "today": "Today",
+  "answer_text_area_hint": "Your thoughts"
```

Comment:
Should this value come from the API? 🤔

Reply:
Oh the conclusion is [here](https://github.com/nimblehq/ic-flutter-taher-toby/pull/81#discussion_r1177327227) 😄